mmdet/apis/test.py

```diff
@@ -11,7 +11,11 @@
 from mmdet.core import tensor2imgs
 
 
-def single_gpu_test(model, data_loader, show=False, out_dir=None):
+def single_gpu_test(model,
+                    data_loader,
+                    show=False,
+                    out_dir=None,
+                    score_thr=0.3):
```

Comment:
During testing, we adopt the score threshold specified in the config file. Here the threshold is only used for visualization, and the variable name `score_thr` can be misleading. Renaming it to `show_score_thr` would be better.

Reply:
I would then set the default to None and use model.test_cfg.score_thr as default in order to be consistent between results and visualization by default.

Reply:
In mmdet/models/detectors/base.py:

```python
score_thr=None,
# ...
if score_thr is None:
    score_thr = self.test_cfg.score_thr
```

Reply:
A higher score is usually preferred for visualization and we hardly use the same threshold as in the config. If we use the same threshold, the plotted image will be messy.